packages/flutter/lib/src/gestures/events.dart

```diff
@@ -1794,8 +1794,19 @@ class PointerScrollEvent extends PointerSignalEvent with _PointerEventDescriptio
     super.position,
     this.scrollDelta = Offset.zero,
     super.embedderId,
+    this.whenIgnored,
   });
 
+  /// Callback called when this event is [ignore]d.
+  final Function? whenIgnored;
+
+  /// Function to call when this event is ignored by the framework.
```

Comment:
nit w.r.t "by the framework": anybody can receive a PointerScrollEvent if they register for it, they should likely be allowed to call this as well?

(This will also need much more detailed docs eventually.)

Reply:
Also not convinced by the "ignore" terminology. We are not telling anybody to ignore this event, we want somebody else to handle it, if they want.

Reply:
> anybody can receive a PointerScrollEvent if they register for it, they should likely be allowed to call this as well?

Definitely, if they want their widget to play well in the cases where they're letting the event go through. Agreed with the documentation.

Reply:
> Also not convinced by the "ignore" terminology. We are not telling anybody to ignore this event, we want somebody else to handle it, if they want.

I'm not attached to "ignore" or any other name in these two PRs. I just couldn't come up with anything that sounded more appropriate than ignored... Maybe `skip`?

(I don't want to call it `allowPropagation` because then it sounds very "the opposite of `stopPropagation` on the web :p)

Reply:
I called it `acknowledge`.